content/en/guides/hosting/data-security/secure-storage-connector.md

```diff
@@ -7,118 +7,236 @@ title: Bring your own bucket (BYOB)
+- [Amazon S3](https://aws.amazon.com/s3/) is an object storage service offering industry-leading scalability, data availability, security, and performance.
+- [Google Cloud Storage](https://cloud.google.com/storage) is a managed service for storing unstructured data at scale.
+- [Azure Blob Storage](https://azure.microsoft.com/products/storage/blobs) is a cloud-based object storage solution for storing massive amounts of unstructured data like text, binary data, images, videos, and logs.
+- S3-compatible storage like [MinIO](https://github.com/minio/minio) hosted in your cloud or infrastructure on your premises.
 
-## Availability matrix
-The following table shows the availability of BYOB across different W&B Server deployment types. An `X` means the feature is available on the specific deployment type.
+The following table shows the availability of BYOB at each scope for each W&B deployment type.
 
-| W&B Server deployment type | Instance level | Team level | Additional information |
-|----------------------------|--------------------|----------------|------------------------|
-| Dedicated cloud | X | X | Both the instance and team level BYOB are available for Amazon Web Services, Google Cloud Platform and Microsoft Azure. For the team-level BYOB, you can connect to a cloud-native storage bucket in the same or another cloud, or even a S3-compatible secure storage like [MinIO](https://github.com/minio/minio) hosted in your cloud or on-prem infrastructure. |
-| SaaS Cloud | Not Applicable | X | The team level BYOB is available only for Amazon Web Services and Google Cloud Platform. W&B fully manages the default and only storage bucket for Microsoft Azure. |
-| Self-managed | X | X | Instance level BYOB is the default since the instance is fully managed by you. If your self-managed instance is in cloud, you can connect to a cloud-native storage bucket in the same or another cloud for the team-level BYOB. You can also use S3-compatible secure storage like [MinIO](https://github.com/minio/minio) for either of instance or team-level BYOB. |
+| W&B deployment type | Instance level                         | Team level | Additional information |
+|----------------------------|----------------------------------------|------------|------------------------|
+| Dedicated Cloud            | &check;<sup><a href="footnote_1">1</a></sup> | &check;          | Instance and team level BYOB are supported for CoreWeave AI Object Storage, AWS S3, GCP Storage, Microsoft Azure Blob Storage, and S3-compatible storage like [MinIO](https://github.com/minio/minio) hosted in your cloud or on-prem infrastructure. |
+| Multi-tenant Cloud                 | Not Applicable                         | &check;<sup><a href="footnote_1">1</a></sup> | Team level BYOB is supported for AWS S3, and GCP Storage. W&B fully manages the default and only storage bucket for Microsoft Azure. Team level BYOB for CoreWeave AI Object Storage is coming soon. |
```

Comment:
Same with this one. consider removing (if not pressured by product to keep this).